Wangscape/tilegen/partition/TilePartitionerPerlin.cpp

```diff
@@ -29,16 +29,16 @@ Reseedable TilePartitionerPerlin::makeCornerModule(const Corners& corners,
     Reseedable ef = noise::module::makeEdgeFavouringMask(1.5, 1.);
     Reseedable corner = ef.blend(stochastic_mask, deterministic);
     // postprocess should be customisable
-    Reseedable postprocess = corner.pow(5.).clamp(0.,std::numeric_limits<float>::infinity());
+    Reseedable postprocess = corner.pow(5.).clamp(0.,std::numeric_limits<double>::infinity());
     return postprocess;
 }
 
-void TilePartitionerPerlin::noiseToAlpha(std::vector<noise::RasterValues<float>>& noise_values,
+void TilePartitionerPerlin::noiseToAlpha(std::vector<noise::RasterValues<double>>& noise_values,
                                          std::vector<sf::Image>& outputs,
                                          size_t resolution) const
 {
-    std::vector<float> weights((int)CORNERS);
-    alpha::AlphaCalculatorMax ac;
+    std::vector<double> weights((int)CORNERS);
+    alpha::AlphaCalculatorLinear ac;
```

Comment:
#51 Only manifests when `AlphaCalculatorLinear` is used. I changed to `AlphaCalculatorLinear` in order to observe the effects of the changes in that use case.

In future the choice of `AlphaCalculator` should be in the options file (#37); at the current stage of the project I don't think it's very important which one is hard-coded.